core/Makefile

```diff
@@ -20,6 +22,45 @@ android:
 	@cp target/i686-linux-android/release/$(libName) $(jniLibs)/x86/$(libName)
 	@cp target/x86_64-linux-android/release/$(libName) $(jniLibs)/x86_64/$(libName)
 
+.PHONY: macos_jni
+macos_jni:
+	@touch src/lib.rs
+	@cargo build --lib --release
+	
+	@echo "Cleaning up old .dylib if they exist"
+	@rm -rf $(jniLibs)/desktop
+	
+	@echo "Moving new .dylib over"
+	@mkdir -p $(jniLibs)/desktop
+
+	@mv target/release/$(macosLibName) $(jniLibs)/desktop/$(macosLibName)
```

Comment:
Any reason you prefer `mv` over `cp`? Still thinking about which I like more

Reply:
I use mv since I can take that library and not have cargo rebuild everything the next time I call the command. So I just thought I might as well just take it rather than copy it over.

Reply:
I imagine `mv` would make cargo rebuild since the target folder has been modified